README.md

```diff
@@ -55,29 +52,15 @@ or install from source:
 python setup.py install
 ```
 
-## Quick start
-
-Before you start, you need to know some experience about [DocArray](https://github.com/jina-ai/docarray).
-`AnnLite` is designed to be used with [DocArray](https://github.com/jina-ai/docarray), so you need to know how to use `DocArray` first.
-
-For example, you can create a `DocArray` with `1000` random vectors with `128` dimensions:
-
-```python
-from docarray import DocumentArray
-import numpy as np
-
-docs = DocumentArray.empty(1000)
-docs.embeddings = np.random.random([1000, 128]).astype(np.float32)
-```
-
-
 ### Index
 
 Then you can create an `AnnIndexer` to index the created `docs` and search for nearest neighbors:
 
 ```python
 from annlite import AnnLite
 
+docs = [{'id': '0', 'embedding': [], 'price': 8}]
```

Comment:
use dummy embedding as the example? `'embedding': np.random.random([128]).astype(np.float32)`